Source/Logging.swift

```diff
@@ -28,69 +26,65 @@ public class RxBluetoothKitLog: ReactiveCompatible {
         case error = 4
     }
 
-    /// Set new log level.
-    /// - Parameter logLevel: New log level to be applied.
-    public static func setLogLevel(_ logLevel: LogLevel) {
-        currentLogLevel = logLevel
-    }
-
-    /// Get current log level.
-    /// - Returns: Currently set log level.
-    public static func getLogLevel() -> LogLevel {
-        return currentLogLevel
-    }
-
-    fileprivate static func tag(with logLevel: LogLevel) -> String {
-        let prefix: String
-
-        switch logLevel {
-        case .none:
-            prefix = "[RxBLEKit|NONE|"
-        case .verbose:
-            prefix = "[RxBLEKit|VERB|"
-        case .debug:
-            prefix = "[RxBLEKit|DEBG|"
-        case .info:
-            prefix = "[RxBLEKit|INFO|"
-        case .warning:
-            prefix = "[RxBLEKit|WARN|"
-        case .error:
-            prefix = "[RxBLEKit|ERRO|"
-        }
-        let time = Date().timeIntervalSinceReferenceDate
-        return prefix + String(format: "%02.0f:%02.0f:%02.0f.%03.f]:",
-                               floor(time / 3600.0).truncatingRemainder(dividingBy: 24),
-                               floor(time / 60.0).truncatingRemainder(dividingBy: 60),
-                               floor(time).truncatingRemainder(dividingBy: 60),
-                               floor(time * 1000).truncatingRemainder(dividingBy: 1000))
-    }
-
-    fileprivate static func log(with logLevel: LogLevel, message: @autoclosure () -> String) {
-        if currentLogLevel <= logLevel {
-            let string = "\(tag(with: logLevel)) \(message())"
-            subject.onNext(string)
```

Comment:
You removed possibility to use `log` observable to observe logs from the library. Please add this to the new `log` method.